expression/integration_test.go

```diff
@@ -4657,6 +4657,10 @@ func (s *testIntegrationSuite) TestDatetimeMicrosecond(c *C) {
 		testkit.Rows("2007-03-28 22:06:28"))
 	tk.MustQuery(`select DATE_ADD('2007-03-28 22:08:28',INTERVAL "-2.-2" MICROSECOND);`).Check(
 		testkit.Rows("2007-03-28 22:08:27.999998"))
+	tk.MustQuery(`select DATE_ADD('2000-04-13 07:17:02',INTERVAL -1465647104 YEAR);`).Check(testkit.Rows("NULL"))
```

Comment:
CI failed.  You need replace the `NULL` to `<nil>`.
And the function is `TestDatetimeMicrosecond`, the test cases should not be included in there.
In fact ,  there is a function `TestDatetimeOverflow` in this file.
You can add the cases in the function, or add a function `TestIssuexxxxx` if you want.

Reply:
Very well, done.